cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -356,7 +358,8 @@ data:
       matches [{ "_SYSTEMD_UNIT": "{{ fluentd_container_runtime_service }}.service" }]
       <storage>
         @type local
-        persistent true
+        persistent false
```

Comment:
why do you remove persistence?
its there to be able to pickup the buffer after a container crash.
otherwise you might loose some logs.